charts/kueue/templates/crd/kueue.x-k8s.io_workloads.yaml

```diff
@@ -8270,6 +8270,18 @@ spec:
                   is associated with. queueName cannot be changed while .status.admission
                   is not null.
                 type: string
+              queueingPolicy:
+                default: Always
+                description: "QueueingPolicy determines if a workload can be admitted
+                  into a queue. Changing the QueueingPolicy to Never does not evict
+                  or suspend workloads that are already running. Possible values are:
```

Comment:
>Changing the QueueingPolicy to Never does not evict or suspend workloads that are already running

Hm, I probably missing something here. 
In https://github.com/kubernetes-sigs/kueue/issues/1091 I think we want to suspend a running job.

Reply:
The user journey is that the user would set the queueing policy to Never and then suspend the job.

But maybe you are right... there is no much reason why Kueue couldn't take the policy as a signal to suspend the Job as well.

But then the name `queueingPolicy` doesn't fit. It would have to be `admissionPolicy`?

@andrewsykim initially said the following:

> One approach I've been thinking about is to add a requeuePolicy field to wworkload spec, with initial values of Always (default) or Never. This seems slightly cleaner than a field to directly suspend the job since we're not introducing new mechanisms to suspend workloads. Thoughts?

The job reconciler is capable of evicting the job if the workload has a signal for that.

Reply:
I think from user point of view the workload and the job are the same thing, no? And I thought in Kueue we want to have keep workload and the job in sync, so any action on workloa == action on job and vice verse, am I wrong here?